domain/src/main/java/org/oppia/domain/oppialogger/OppiaLogger.kt

```diff
@@ -0,0 +1,123 @@
+package org.oppia.domain.oppialogger
+
+import org.oppia.app.model.EventLog
+import org.oppia.app.model.EventLog.EventAction
+import org.oppia.domain.oppialogger.analytics.AnalyticsController
+import javax.inject.Inject
+
+/** Logger that handles exceptions, crashes, events, and console logging. */
```

Comment:
Is this doc string true? It seems to only be for logging events currently.

Reply:
The follow up PRs will add exceptions , crashes, and console logging. That's why I added it right now. I can remove the others now and add them with each PR